### PR TITLE
docstring for `@time_imports`: explain what is shown (it's not cumulative)

### PR DESCRIPTION
**Pull request by tfiers**
Re https://github.com/JuliaLang/julia/pull/45861#issuecomment-1380220029

### CODE REVIEW

File: stdlib/InteractiveUtils/src/macros.jl [hunk 1]
```diff
@@ -360,6 +360,9 @@ See also: [`code_native`](@ref), [`@code_llvm`](@ref), [`@code_typed`](@ref) and
 A macro to execute an expression and produce a report of any time spent importing packages and their
 dependencies. Any compilation time will be reported as a percentage, and how much of which was recompilation, if any.
 
+One line is printed per package. The duration shown is the time to import that package itself
+(and not any of its dependencies).
```
Comment (IanButterworth) [inline stdlib/InteractiveUtils/src/macros.jl:364]:
```suggestion
One line is printed per package or package extension. The duration shown is the time to import that package itself, not including the time to load any of its dependencies.
```

**Comment by gbaraldi** [comment]
Could we print the total value when using `@time_imports` so it's clearer even, and I also imagine that people might want that.

**Comment by IanButterworth** [comment]
> Could we print the total value when using @time_imports so it's clearer even, and I also imagine that people might want that.

It's not straightforward. See https://github.com/JuliaLang/julia/pull/47101

**Comment by gbaraldi** [comment]
I see, couldn't we at least add up all the times, and also figure out what that extra time is :).

**Pull request merged**